### ISSUE TITLE
Add auto-responder to contact form

### ISSUE THREAD

**Issue by kkilbey** [BCcampus/bcc-sage#305]
## Submit a feature request or bug report

- [x] This is a feature request
- [ ] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/BCcampus/bcc-sage/issues)
- [x] You cleared your Web Browsers cache to ensure all new styling and images were loaded. 

---

**What is the current behavior?**

There is currently no auto-responder on the contact form.

**What is the expected or desired behavior?**

When someone submits a message through the contact form, an auto-responder sends an email notifying them that their message has been received.

Blocked by issue #300. We are currently receiving lots of spam through the contact form, and we don't want to send auto-responses to spammers, as that would provide them with our email address. Once we have a stronger captcha in place to filter out spam, we can add the auto-responder.

**Comment by paulagaube** [comment]
Kelsey and I configured the "Mail (2)" autoresponder email on the "Website Contact Form" and tweaked it to our satisfaction, then unchecked the option to "Use Mail (2)" until the captcha issue is resolved. 

Dashboard > Contact > Contact Forms > Mail tab 
https://bccampus.ca/wp-admin/admin.php?page=wpcf7&post=8066&active-tab=0

**Comment by kkilbey** [comment]
Since we resolved the excessive spam issue, I have enabled the autoresponder on the "Website Contact Form." Changed [your-name] to [first-name] and tested the autoresponder, which is working as planned. Closing this issue.
